WORKSPACE

```diff
@@ -29,6 +29,16 @@ load("//:internal_dev_deps.bzl", "rules_python_internal_deps")
 
 rules_python_internal_deps()
 
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
```

Comment:
This was simply never called before and now updating the rules_go version seems to have a lot of issues with things like nogo targets missing 🤷